core/src/main/scala/com/salesforce/op/filters/Summary.scala

```diff
@@ -59,7 +80,10 @@ case object Summary {
    */
   def apply(preppedFeature: ProcessedSeq): Summary = {
     preppedFeature match {
-      case Left(v) => Summary(v.size, v.size, v.size, 1.0)
+      case Left(v) =>
+        val textLenMoments = MomentsGroup.sum(v.map(x => Moments(x.length.toDouble)))
+        val tokenDistribution = TextStats(v.groupBy(identity).mapValues(_.size))
```

Comment:
This looks weird to me, why not just `v.map(x => x -> x.length).toMap`?

Reply:
Oh nm, you want the counts of each token - I got it.

Reply:
what if there are repeats? but also you should not do a mapValues. So 
`v.groupBy(identity).map( t => t -> t.size )`

Reply:
I think `groupBy(identity)` takes care of the repeats.